embulk-core/src/main/java/org/embulk/exec/NullReporterPlugin.java

```diff
@@ -0,0 +1,46 @@
+package org.embulk.exec;
+
+import org.embulk.config.ConfigSource;
+import org.embulk.config.Task;
+import org.embulk.config.TaskSource;
+import org.embulk.spi.ReporterCloseable;
+import org.embulk.spi.ReporterPlugin;
+import org.embulk.spi.util.AbstractReporter;
+import org.embulk.spi.util.Reporters;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+import java.util.Map;
+
+public class NullReporterPlugin
```

Comment:
Thank you. I will. 

BulkLoader needs to instantiate reporter plugin and reporters. Since embulk-core doesn't depends on embulk-standards, default reporter needs to be put on embulk-core, in particular, on `org.embulk.exec`.

As talked to you directly, I will put `StdoutReporterPlugin` to `org.embulk.exec`. and then, `NullReporterPlugin` will be moved to embulk-standard. Because it's good to use stdout reporter by default.